src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -43,76 +45,88 @@
+})
+@Since("2.1, 2.5.3 (block break event), 2.7 (experience change event), INSERT VERSION (breeding event)")
+@Events({"experience spawn", "break / mine", "experience change", "entity breeding"})
 public class ExprExperience extends SimpleExpression<Experience> {
+
 	static {
 		Skript.registerExpression(ExprExperience.class, Experience.class, ExpressionType.SIMPLE, "[the] (spawned|dropped|) [e]xp[erience] [orb[s]]");
 	}
-	
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
-		if (!getParser().isCurrentEvent(ExperienceSpawnEvent.class, BlockBreakEvent.class, PlayerExpChangeEvent.class)) {
-			Skript.error("The experience expression can only be used in experience spawn, block break and player experience change events");
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		if (!getParser().isCurrentEvent(ExperienceSpawnEvent.class, BlockBreakEvent.class, PlayerExpChangeEvent.class, EntityBreedEvent.class)) {
+			Skript.error("The experience expression can only be used in experience spawn, block break, player experience change and entity breeding events");
 			return false;
 		}
 		return true;
 	}
 	
 	@Override
 	@Nullable
-	protected Experience[] get(final Event e) {
-		if (e instanceof ExperienceSpawnEvent)
-			return new Experience[] {new Experience(((ExperienceSpawnEvent) e).getSpawnedXP())};
-		else if (e instanceof BlockBreakEvent)
-			return new Experience[] {new Experience(((BlockBreakEvent) e).getExpToDrop())};
-		else if (e instanceof PlayerExpChangeEvent)
-			return new Experience[] {new Experience(((PlayerExpChangeEvent) e).getAmount())};
+	protected Experience[] get(Event event) {
+		if (event instanceof ExperienceSpawnEvent)
+			return new Experience[] {new Experience(((ExperienceSpawnEvent) event).getSpawnedXP())};
+		else if (event instanceof BlockBreakEvent)
+			return new Experience[] {new Experience(((BlockBreakEvent) event).getExpToDrop())};
+		else if (event instanceof PlayerExpChangeEvent)
+			return new Experience[] {new Experience(((PlayerExpChangeEvent) event).getAmount())};
+		else if (event instanceof EntityBreedEvent)
+			return new Experience[] {new Experience(((EntityBreedEvent) event).getExperience())};
 		else
 			return new Experience[0];
 	}
 	
 	@Override
 	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
+	public Class<?>[] acceptChange(ChangeMode mode) {
 		switch (mode) {
-			case ADD:
-			case DELETE:
-			case REMOVE:
-			case REMOVE_ALL:
-				return new Class[] {Experience[].class, Number[].class};
 			case SET:
-				return new Class[] {Experience.class, Number.class};
+			case DELETE:
 			case RESET:
-				return null;
+				return CollectionUtils.array(ExprExperience.class, Integer.class);
+			case ADD:
+			case REMOVE:
+				return CollectionUtils.array(ExprExperience[].class, Integer[].class);
```

Comment:
```suggestion
				return CollectionUtils.array(Experience.class, Integer.class);
			case ADD:
			case REMOVE:
				return CollectionUtils.array(Experience[].class, Integer[].class);
```
This isn't supposed to be ExprExperience, right?